monai/apps/detection/networks/retinanet_network.py

```diff
@@ -0,0 +1,376 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import math
+from typing import Callable, Dict, List, Sequence, Union
+
+import torch
+from torch import Tensor, nn
+
+from monai.networks.blocks.backbone_fpn_utils import _resnet_fpn_extractor
+from monai.networks.layers.factories import Conv
+from monai.networks.nets import resnet
+from monai.utils import ensure_tuple_rep, look_up_option, optional_import
+from monai.utils.misc import issequenceiterable
+
+_validate_trainable_layers, _ = optional_import(
+    "torchvision.models.detection.backbone_utils", name="_validate_trainable_layers"
+)
+
+
+class RetinaNetClassificationHead(nn.Module):
+    """
+    A classification head for use in RetinaNet.
+
+    This head takes a list of feature maps as inputs, and outputs a list of classification maps.
+    Each output map has same spatial size with the corresponding input feature map,
+    and the number of output channel is num_anchors * num_classes.
+
+    Args:
+        in_channels: number of channels of the input feature
+        num_anchors: number of anchors to be predicted
+        num_classes: number of classes to be predicted
```

Comment:
missing doc-string for `spatial_dims` and `prior_probability`(also missing typehint).

Thanks.